app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb

```diff
@@ -8,9 +8,11 @@
   <li>science if you want to teach primary</li>
 </ul>
 
-<p>You may be able to show you meet the standard in another way if you do not have these GCSEs.</p>
+<p>You may be able to show you meet the standard in another way if you do not have GCSE's.</p>
```

Comment:
```suggestion
<p>You may be able to show you meet the standard in another way if you do not have GCSEs.</p>
```

I think GCSEs shouldn't have an apostrophe here.

Reply:
Yep looks that way elsewhere good catch